Code Quality/Tools.md

```diff
@@ -232,8 +232,8 @@ npm i -D concurrently husky lint-staged ng-lint-staged stylelint stylelint-prett
 
 Some notes:
 
--   it is important to run `tsc` on all files because changes in staged files can affect compilation of unmodified files
--   `tsc` is run on both the application `tsconfig` files and tests `tsconfig` files
--   `concurrently` speeds up things by running `tsc` checks in parallel
--   `prettier --write` is run separately for `.ts` and other files in order to prevent any possible race conditions before running TSLint (via `lint:ng`) and Prettier
--   `ng-lint-staged` is required because `ng lint` does not accept the list of files as provided by `lint-staged`, so some transformations are necessary
+- it is important to run `tsc` on all files because changes in staged files can affect compilation of unmodified files
+- `tsc` is run on both the application `tsconfig` files and tests `tsconfig` files
+- `concurrently` speeds up things by running `tsc` checks in parallel
+- `prettier --write` is run separately for `.ts` and other files in order to prevent any possible race conditions before running TSLint (via `lint:ng`) and Prettier
+- `ng-lint-staged` is required because `ng lint` does not accept the list of files as provided by `lint-staged`, so some transformations are necessary
```

Comment:
I think we also no longer use `ng-lint-staged` because `eslint` works with regular `lint-staged`. I will take a more detailed look into this so you don't have to change it right now